### PR TITLE
Add set_dri_name and call from entrypoint if no DRI_NAME is provided

### LINKED ISSUE #441: Script for automatic GPU selection

### PR DESCRIPTION
**Pull request by dpascualhe**

### CODE REVIEW

**Comment by dpascualhe** [comment]
Split from https://github.com/JdeRobot/RoboticsAcademy/pull/2754

**Review by javizqh — CHANGES_REQUESTED** [review]
Can you export the vendor name, please?

File: scripts/set_dri_name.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        fi
+
+        # Search for the vendor in the GPU list from lspci
+        gpu_info=$(echo "$gpu_list" | grep -i "$vendor" | head -n 1)
+        if [ -n "$gpu_info" ]; then
+            bus=$(echo "$gpu_info" | cut -d' ' -f1)
+            device=$(ls /sys/bus/pci/devices/0000:$bus/drm | grep card)
+            if [ -n "$device" ]; then
+                # Set the DRI_NAME environment variable to the card name (e.g., card0, card1)
+                export DRI_NAME="$device"
```
Comment (javizqh) [inline scripts/set_dri_name.sh:33]: Here export it as DRI_VENDOR

Comment (dpascualhe) [reply scripts/set_dri_name.sh:33]: Done

**Review by javizqh — APPROVED** [review]

**Pull request merged**